package.json

```
@@ -18,7 +18,7 @@
     "packageManager": "pnpm@7.14.2",
     "scripts": {
         "copy-scripts": "mkdir -p frontend/dist/ && pnpm copy-scripts:array && pnpm copy-scripts:recorder",
```

Could simplify all this to just:
```suggestion
        "copy-scripts": "mkdir -p frontend/dist/ && ./bin/copy-posthog-js",
```